src/v2/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx

```diff
@@ -37,12 +41,24 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
     slug: artist.slug,
   }
 
+  const metric = getSupportedMetric(me?.lengthUnitPreference)
+  const filters: ArtworkFiltersState = {
+    metric,
+    ...match.location.query,
+  }
+
   return (
     <ArtworkFilterContextProvider
       aggregations={aggregations}
       counts={artist.counts as Counts}
-      filters={match.location.query}
-      onChange={updateUrl}
+      filters={filters}
+      onChange={filterState => {
+        updateUrl(filterState, {
+          defaultValues: {
+            metric,
+          },
```

Comment:
These changes are necessary to ensure that the user’s preferred metric is not displayed in the url by default if it differs from the default metric.

### Example
 * [Default metric](https://github.com/artsy/force/blob/9dc6d876e8c7073aba6f3742585faebea2460bd5/src/v2/Components/ArtworkFilter/Utils/metrics.ts#L3) is `CM`
 * User’s preferred metric is `IN`
 * If there were no such changes, the url would contain `metric=IN` (something like this: `artsy.net/artist/banksy/works-for-sale?metric=IN`)

Reply:
I think it would probably be fine to display the metric in the URL 🤷 I don't feel strongly about it, but if this adds to the amount of code we have to add to all of our artwork grids and adds a bit of complexity for future developers, I might suggest we just show the metric in the address bar.

Reply:
Worth noting that we also want users to be able to toggle back and forth on-page, and we plan to save that unit (`in/cm`) on individual saved alerts as well. 

Here's how I'm thinking of the actions we need to support:
- Preference is stored on user and fetched on page load
- User can still toggle to the other metric that is not their preference
- User can save an alert with either unit

I don't have a strong opinion about in vs. out of URL as long as all of this works 🤷‍♂️

Reply:
I agree with @mdole that metric should be in the URL

Reply:
Sounds good!